### DISCUSSION TITLE
createAvatar [object, object] in Vue.

### DISCUSSION THREAD

**Discussion by Tomo5524** [dicebear/dicebear#156] [Q&A] [answered]
I am trying to preview avatar but when I check img, src refers to [object, object]. I tried JSON.stringify() but did not work. Console.log gives the right URL with which I can get the desired avatar. The only problem is I cannot display the image of avatar as its src is [object, object].

**Comment by FlorianKoerner** [comment, marked as answer]
Do you have a code example? Here's what works for me. But I'm not sure if this answers your question.

```vue
<template>
  <img alt="Avatar" v-bind:src="avatar">
</template>

<script>
import { createAvatar } from "@dicebear/avatars";
import * as style from "@dicebear/open-peeps";

export default {
  name: "App",
  computed: {
    avatar: () => {
      return createAvatar(style, {
        dataUri: true,
        size: 128
     });
    }
  }
};
</script>
```

CodeSandbox Example:
https://codesandbox.io/s/dry-voice-x3wv5?file=/src/App.vue

**Reply by Tomo5524** [reply]
I just realized that I was using curly brackets in v-bind. Now it is working now! great API!!!
